app/my_api/v2/auth.py

```diff
@@ -181,11 +181,56 @@ def post(self):
                 access_token = create_access_token(identity=email)
                 refresh_token = create_refresh_token(identity=email)
                 return {'Message': 'Logged in as {}'.format(email),
-                        'Access Token': access_token}, 201
+                        'Access Token': access_token}, 200
             else:
                 return {'Message': 'Invalid credentials'}, 403
 
         except (Exception, psycopg2.DatabaseError) as error:
             cur.execute("rollback;")
             print(error)
             return {'Message': 'current transaction is aborted'}, 500
+
+
+class ResetPassword(Resource):
+    """docstring for ResetPassword."""
+    parser = reqparse.RequestParser()
+    parser.add_argument(
+        'email',
+        type=str,
+        required=True,
+        help="Email field empty"
+    )
+    parser.add_argument(
+        'password',
+        type=str,
+        required=True,
+        help="Password field empty"
+    )
+    parser.add_argument(
+        'passwordconfirm',
+        type=str,
+        required=True,
+        help="Confirm password field empty"
+    )
+
+    @classmethod
+    def put(self):
```

Comment:
On the specification, it is a POST request for reset password. Consider updating the same